app/components/contacts/CreateNewContact.js

```diff
@@ -162,7 +162,7 @@ class CreateNewContact extends Component<Props, State> {
     if (nicknameRegex.test(nickname)) {
       return 'Nickname is missing or invalid';
     }
-    const addressRegex = /\b[a-zA-Z0-9]{64}\b/;
+    const addressRegex = /\b[a-zA-Z0-9]{42}\b/;
```

Comment:
if we want restrict addresses to those starting with '0x' we need to add it to the beginning of the regex

Reply:
made the changes and now updateTxContact loops over all accounts + all transactions